addons/mail/wizard/mail_compose_message.py

```diff
@@ -1093,8 +1093,7 @@ def _process_mail_values_state(self, mail_values_dict):
         blacklist_ids = self._get_blacklist_record_ids(mail_values_dict, recipients_info)
         optout_emails = self._get_optout_emails(mail_values_dict)
         done_emails = self._get_done_emails(mail_values_dict)
-        # in case of an invoice e.g.
-        mailing_document_based = self.env.context.get('mailing_document_based')
```

Comment:
Does this mean this key is no longer used?

I found an occurrence in "industry_fsm" in the enterprise repo, should be cleaned too?